spec/menu_spec.rb

```diff
@@ -0,0 +1,52 @@
+require './lib/menu'
+
+describe Menu do
+
+  describe '#menu' do
+    it "defines the method" do
+      expect(subject).to respond_to(:menu)
```

Comment:
Remove respond_to tests once you have a test for the actual functionality of a method. 
Good grouping of tests by method